appium/webdriver/webdriver.py

```diff
@@ -523,9 +523,25 @@ def update_settings(self, settings):
 
     @property
     def device_time(self):
-        """Returns the date and time from the device
+        """Returns the date and time from the device.
         """
-        return self.execute(Command.GET_DEVICE_TIME, {})['value']
+        return self.execute(Command.GET_DEVICE_TIME_GET, {})['value']
+
+    def get_device_time(self, format=None):
+        """Returns the date and time from the device. (Only available since Appium 1.11.0)
+
+        :Args:
+         - format - The default format is `YYYY-MM-DDTHH:mm:ssZ`, which complies to ISO-8601
+           The set of format specifiers. Read https://momentjs.com/docs/ to get
+           the full list of supported datetime format specifiers.
```

Comment:
please also add that if unset then the result of `device_time` is returned by default

Reply:
Done by https://github.com/appium/python-client/pull/312/commits/771324b0d9d36a4da3d7ee8bce3a4c0999f6d58d